docs/theory.rst

```diff
@@ -189,6 +189,79 @@ The example below shows what happens with ``n_steps`` = 4.
 The minimum number of steps is 3; if ``n_steps`` is not set by the user, or if it is set to a value less than 3, the code automatically changes ``n_steps`` to the default value.
 
 
+Factoring the Gaussian Filter
+-----------------------------
+
+Per equation (13) of `Grooms et al. (2021) <https://doi.org/10.1029/2021MS002552>`_, the data to be filtered can be thought of as a vector :math:`\vec{f}` that can be expanded in an orthonormal basis of eigenfunctions of the discrete Laplacian
+
+.. math:: \vec{f} = \sum_i \hat{f}_i\vec{q}_i
+
+where :math:`\vec{q}_i` are the orthonormal basis vectors.
+Each basis vector is associated with a wavenumber :math:`k_i` and a length scale :math:`2\pi/k_i`.
+The filtered data can also be expanded in the same basis
+
+.. math:: \sum_i \hat{f}_ip(k_i^2)\vec{q}_i
+
+The polynomial :math:`p` approximates the target filter.
+If you filter the data :math:`N` times using the same filter, then the result has the following expansion
+
+.. math:: \sum_i \hat{f}_i(p(k_i^2))^N\vec{q}_i
+
+For a Gaussian filter, the target filter is
+
+.. math:: p(k^2) \approx g_L(k) = \text{exp}\left\{-\frac{L^2}{24}k^2\right\}
+
+where :math:`L` is the ``filter_scale``.
+The Gaussian target has the nice property that :math:`g_L^N = g_{\sqrt{N}L}`, i.e. if you apply the Gaussian filter :math:`N` times with scale :math:`L`, the result is the same as if you applied the Gaussian filter once with scale :math:`\sqrt{N}L`.
+
+The way that we sometimes use this is that instead of filtering once with scale :math:`L`, we filter :math:`N` times with scale :math:`L/\sqrt{N}`.
+Unfortunately, because we're using a polynomial approximation rather than an exact Gaussian filter, filtering once with scale :math:`L` is not exactly the same as filtering :math:`N` times with scale :math:`L/\sqrt{N}`.
+The difference between the filtered field obtained using a single Gaussian versus :math:`N` factored Gaussians is exactly
+
+.. math:: \sum_i \hat{f}_i(p_1(k_i^2) - (p_N(k^2))^N)\vec{q}_i
+
+where :math:`p_1(k^2)` is the polynomial that approximates :math:`g_{L}(k)` and :math:`p_N(k^2)` is the polynomial that approximates :math:`g_{L/\sqrt{N}}(k)`.
+We can bound the 2-norm of this error as follows.
+For :math:`p_1` we have the error
+
+.. math:: g_L(k) = p_1(k^2) + e_1(k)
+
+and for :math:`p_N` we have the error
+
+.. math:: g_{L/\sqrt{N}}(k) = p_N(k^2) + e_N(k)
+
+The aforementioned property of the Gaussian implies that
+
+.. math:: g_{L}(k) = (p_N(k)^2 + e_N(k))^N = (p_N(k^2))^N + N (p_N(k^2))^{N-1} e_N(k) + \ldots + (e_N(k))^N
+
+(using the binomial expansion.)
+Subtracting gives us an expression for the difference between the polynomial approximation with scale :math:`L` and the factored approximation using :math:`N` filters each with scale :math:`L/\sqrt{N}`:
```

Comment:
This is optional, but rather than saying "subtracting" we could say "subtracting (1) from (2)" via equation cross-references and labels, see [here](https://developer.lsst.io/v/DM-5063/docs/rst_styleguide.html#math).

Reply:
Good idea, and wasn't too hard to implement.